src/components/Footer/styled.js

```diff
@@ -0,0 +1,14 @@
+import Typography from '@mui/material/Typography';
+import { styled } from "@mui/material/styles";
+
+export const StyledFooter = styled(Typography)`
+display: flex;
```

Comment:
We shouldn't use `display: flex` for text elements, previously it was for `footer` element, and it is not text element. To fix that you should wrap it with `Box` component and place those styles to that component. Typography will be without any styling.